MdeModulePkg/Library/DxeCorePerformanceLib/DxeCorePerformanceLib.c

```diff
@@ -228,16 +230,18 @@ InternalGetSmmPerfData (
 {
   EFI_STATUS                               Status;
   UINT8                                    *SmmBootRecordCommBuffer;
-  EFI_SMM_COMMUNICATE_HEADER               *SmmCommBufferHeader;
+  EFI_MM_COMMUNICATE_HEADER                *SmmCommBufferHeader;      // MU_CHANGE - Standalone MM Perf Support

```

Comment:
are these 2 headers technically identical? if so, do you want to update the variable name as well?

Reply:
I was going to hold off on some changes like that to reduce `MU_CHANGE` lines, but I'll go for it to keep things clean.